src/hooks/useTenantTimezone.js

```diff
@@ -0,0 +1,84 @@
+import { userLocaleConfig, useStripes, useOkapiKy } from '@folio/stripes/core';
+import { useQuery } from 'react-query';
+
+export function getQueryWarning(tenantTimezone, userTimezone) {
+  if (tenantTimezone === userTimezone) {
+    return null;
+  }
+  if (!tenantTimezone || !userTimezone) {
+    return null;
+  }
+
+  // TODO: add a warning here!, and also use this...
+  return 'a warning should go here! TODO [UIPQB-155]';
```

Comment:
handled exposing logic of this warning as part of this ticket, but actual language and display comes later